jbrowse/resources/external/minimalSession.json

```diff
@@ -49,7 +49,7 @@
           }
         }
       },
-      "variantDisplays": [
+      "metadata": [
```

Comment:
OK, so this is an improvement, but isnt it a problem to pollute the top-level namespace like this? We should at least do:

metadata: {
   variantDisplayConfig: {....}
}

Note: I suggest we take this opportunity to rename the property to be more in line with our plugin too. Maybe 'extendedVariantDisplayConfig' would be even better?

Reply:
Name and location within metadata updated.